pkg/peer/webrtc.go

```diff
@@ -110,6 +110,7 @@ func (p *Peer[ID]) onConnectionStateChanged(state webrtc.PeerConnectionState) {
 
 	switch state {
 	case webrtc.PeerConnectionStateFailed, webrtc.PeerConnectionStateDisconnected, webrtc.PeerConnectionStateClosed:
+		p.logger.Debug("Ending call since we're in state: %v", state)
```

Comment:
We probably don't need this one since we log the state as `Info` just 2 lines above this 🙂